base/src/main/java/io/spine/validate/ValidationException.java

```diff
@@ -66,7 +66,10 @@ public String toString() {
 
         String violationContent = constraintViolations.isEmpty()
```

Comment:
Please move the expression to the next line and then fix the formatting, so that everything fits. Maybe use a static import. `Collectors.toList()` should be properly aligned.